js/dialog.js

```diff
@@ -43,15 +43,15 @@ function attackTraitsHTML(cost, damage, type) {
   }
   const htmlList = [];
   if (type) {
-    htmlList.push(`Type: ${type}`);
+    htmlList.push(`<span><strong>Type:</strong> ${type}</span>`);
   }
   if (cost) {
-    htmlList.push(`Cost: ${cost}`);
+    htmlList.push(`<span><strong>Cost:</strong> ${cost}</span>`);
   }
   if (damage) {
-    htmlList.push(`Damage: ${damage}`);
+    htmlList.push(`<span><strong>Damage:</strong> ${damage}</span>`);
```

Comment:
this will conflict w o5v3's change so depending on who goes first the other will have to update this spot